MFVNeutralino/python/TriggerFilter_cfi.py

```diff
@@ -32,12 +37,52 @@
     "HLT_HT650_DisplacedDijet60_Inclusive_v*",
     ]
 
+#For 2017 :
 lepton_paths = [
-    #"HLT_Ele35_WPTight_Gsf_v*",
-    #"HLT_Ele115_CaloIdVT_GsfTrkIdT_v*",
-    #"HLT_Ele50_CaloIdVT_GsfTrkIdT_PFJet165_v*",
-    "HLT_IsoMu27_v*",
-    #"HLT_Mu50_v*",
+     "HLT_Ele35_WPTight_Gsf_v*",
+     "HLT_Ele115_CaloIdVT_GsfTrkIdT_v*",
+     "HLT_Ele50_CaloIdVT_GsfTrkIdT_PFJet165_v*",
+     "HLT_IsoMu27_v*",
+     "HLT_Mu50_v*"
+ ]
+
+muon_paths = [
+     "HLT_IsoMu27_v*",
+     "HLT_Mu50_v*"
+]
+
+muoniso_paths = [
+     "HLT_IsoMu27_v*",
+]
+
+electron_paths = [
+     "HLT_Ele35_WPTight_Gsf_v*",
+     "HLT_Ele115_CaloIdVT_GsfTrkIdT_v*",
+     "HLT_Ele50_CaloIdVT_GsfTrkIdT_PFJet165_v*",
+]
+#For 2018 :
+#lepton_paths = [
+#    "HLT_Ele32_WPTight_Gsf_v*",
+#    "HLT_Ele115_CaloIdVT_GsfTrkIdT_v*",
+#    "HLT_Ele50_CaloIdVT_GsfTrkIdT_PFJet165_v*",
+#    "HLT_IsoMu27_v*",
+#    "HLT_Mu50_v*"
+#]
+
+#For 2016 :
+# lepton_paths = [
+#     "HLT_Ele27_WPTight_Gsf_v*",
+#     "HLT_Ele115_CaloIdVT_GsfTrkIdT_v*",
+#     "HLT_Ele50_CaloIdVT_GsfTrkIdT_PFJet165_v*",
+#     "HLT_IsoMu27_v*",
+#     "HLT_Mu50_v*"
+# ]
```

Comment:
Do we just uncomment these when dealing with other years? @abbywarden  @pkotamnives

Reply:
This has been fixed for me in my forked repo but has not yet been propagated to this pull request.

Reply:
Thanks @abbywarden! @pkotamnives and @duquale should take note then, that the triggers for the other years may not be in place yet!